### ISSUE TITLE
The "alwaysArray" option doesn't apply to "exclude"

### ISSUE THREAD

**Issue by davidmz** [nepsilon/search-query-parser#27]
See example at: https://runkit.com/davidmz/5c86c4ecc72b580012f955eb

I expect that it should be `exclude: {in: ["space"]}` but got `exclude: {in: "space"}`

**Comment by nepsilon** [comment]
Thanks for the report. Would you want to provide a PR handling this case?

**Comment by nepsilon** [comment]
Thanks @rschick 👍 Merged and published as v1.5.1.

**Comment by LordMike** [comment]
Hey - with 1.6.0, I'm seeing the following:

```
searchQueryParse('-banana', {tokenize: true, offsets: false})
{
    "exclude": {
        "text": "banana"
    }
}

searchQueryParse('-banana', {tokenize: true, offsets: false, alwaysArray: true})
{
    "exclude": {
        "text": "banana"
    }
}
```

But doing multiple values works:
```
searchQueryParse('-banana -baddie', {tokenize: true, offsets: false})
{
    "exclude": {
        "text": [
            "banana",
            "baddie"
        ]
    }
}
```
